### PR TITLE
Rename file rust-ast-full-test.cc to rust-ast.cc

### PR DESCRIPTION
**Pull request by Parthib314**
* Fixes issue  #1765

**Rename the `rust-ast-full-test.cc` to `rust-ast.cc` and also update it on `Make-lang.in`**

### CODE REVIEW

**Review by CohenArthur — APPROVED** [review]
LGTM :) If you keep making pull-requests, please consider starting to add DCO signoff to your commits :D As the aim is to merge all of these commits into GCC, small contributions without DCO-signoff are allowed, but multiple contributions require it. No worries for now :) Thanks for working on this!

**Comment by Parthib314** [comment]
> LGTM :) If you keep making pull-requests, please consider starting to add DCO signoff to your commits :D As the aim is to merge all of these commits into GCC, small contributions without DCO-signoff are allowed, but multiple contributions require it. No worries for now :) Thanks for working on this!

Ok, I'll keep it in mind. Thanks

**Pull request merged**
